node/src/server-modules/GlideJson.ts

```diff
@@ -400,4 +400,58 @@ export class GlideJson {
 
         return _executeCommand<ReturnTypeJson<GlideString>>(client, args);
     }
+
+    /**
+     * Returns the length of the JSON string value stored at the specified `path` within
+     * the JSON document stored at `key`.
+     *
+     * @param client - The client to execute the command.
+     * @param key - The key of the JSON document.
+     * @param options - (Optional) Additional parameters:
+     * - (Optional) path - The path within the JSON document, Defaults to root if not provided.
```

Comment:
Shouldn't this be @param?

Reply:
We've decided to put `path` arguments in `options` for all commands that applies for the node client, and this line is a sub item of the options param